bokehjs/src/lib/models/renderers/renderer.ts

```diff
@@ -35,6 +58,12 @@ export abstract class RendererView extends View implements visuals.Renderable {
     super.connect_signals()
     const {x_range_name, y_range_name} = this.model.properties
     this.on_change([x_range_name, y_range_name], () => this._initialize_coordinates())
+    const {group} = this.model
+    if (group != null) {
+      this.on_change(group.properties.visible, () => {
+        this.model.visible = group.visible
+      })
```

Comment:
What if the group itself changes?

Reply:
Nothing, the same as everywhere else we rely on transitive properties and do not implement special handling. I intentionally left this like this, because I'm working on a generic mechanism that will handle this case (in particular remove old connections).